README.md

```diff
@@ -312,20 +316,63 @@ A few additional links to some typical early `Jupyter` topics:
 * [Closing running Jupyter notebook servers](https://github.com/jupyter/notebook/issues/2844)
 * [Checkpoints and autosave](https://groups.google.com/forum/#!topic/jupyter/DGCKE5fS4kQ)
 
-### Continuous Integration
+#### Command-line Interface (CLI)
+
+Python's ecosystem offers several ways to tackle command-line interfaces. The traditional standard method is to use
+the `argparse` module that is part of the standard library. This can be complemented by something like `argparsetree`
+for larger and more complex command-line applications.
+
+Here we have chosen to use [Click](https://click.palletsprojects.com/) instead, which allows us to define our CLI via
+decorated functions in a neat and compact way. Other potential alternatives could
+be [docopt](https://docopt.readthedocs.io/) or [Invoke](https://www.pyinvoke.org/).
```

Comment:
is `docopt` the only lower case tool here as they also refer to themselves more with lower case?

Reply:
yes